tests/foreman/api/test_contentmanagement.py

```diff
@@ -1225,6 +1225,25 @@ def test_positive_sync_container_repo_end_to_end(
             )
             assert result.status == 0
 
+        # Inspect the images with skopeo (BZ#2148813)
+        result = module_capsule_configured.execute('yum -y install skopeo')
+        assert result.status == 0
+
+        target_sat.api.LifecycleEnvironment(
+            id=function_lce.id, registry_unauthenticated_pull='true'
+        ).update(['registry_unauthenticated_pull'])
+
+        skopeo_cmd = 'skopeo --debug inspect docker://'
+        for path in repo_paths:
+            result = module_capsule_configured.execute(
+                f'{skopeo_cmd}{target_sat.hostname}/{path}:latest'
```

Comment:
Looking at the PRT failure `level=debug msg="No credentials matching hostname/owxmaghyv-dhgdgwylp-ejgpbagrdl-hhrgdtkwfy-gvsq` it seems it failed due to authentication, so not sure if LCE update for `registry_unauthenticated_pull` worked for capsule or might be capsule sync required after an LCE update?

Reply:
Looking into robottelo.log at the PRT run, the same msg was given for the SAT (line 1238), where the status was 0 and stdout provided valid output (it succeeded). While for the CAPS (line 1242) the status was 1 and no stdout (it failed). Locally, both pass well.